iroha/src/sumeragi.rs

```diff
@@ -332,6 +333,12 @@ impl Sumeragi {
                         error_result
                     )
                 });
+            self.start_commit_countdown(
+                voting_block.clone(),
```

Comment:
Just to double check, do we need to clone a `voting_block` here?

Reply:
Yes, I double checked, seems correct.